message-me/app/controllers/chatrooms_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ChatroomsController < ApplicationController
+  before_action :require_user
```

Comment:
I hate the name `require_user`. Not clear on what this is doing.

Reply:
I changed it to validate_user_logged_in